### PR TITLE
Kinetic Accelerator Tweaks

### PR DESCRIPTION
**Pull request by BurgerLUA**
# Overview
The results are in: Science doesn't always provide mining with kinetic accelerators. As a result of these findings, high level kinetic accelerators now have increased drop chances in the warehouse and in abandoned crates.

Also, kinetic accelerators no longer dig holes.

### CODE REVIEW

**Comment by Pacmandevil** [comment]
>kinetic accelerators no longer dig holes.

As in like, the "shovel in the ground" holes, or are they just pew pew valid guns now.

**Comment by BurgerLUA** [comment]
shovel in the ground holes

File: code/modules/mining/abandonedcrates.dm [hunk 1]
```diff
@@ -23,16 +23,10 @@
 /obj/structure/closet/crate/secure/loot/proc/generate_loot()
 	var/loot = rand(1, 100)
 	switch(loot)
-		if(1 to 5) // Common things go, 5%
```
Comment (Sindorman) [inline code/modules/mining/abandonedcrates.dm]: Why did you remove these?

File: code/modules/mining/abandonedcrates.dm [hunk 2]
```diff
@@ -23,16 +23,10 @@
...
-			new/obj/item/weapon/reagent_containers/food/drinks/bottle/rum(src)
-			new/obj/item/weapon/reagent_containers/food/drinks/bottle/whiskey(src)
-			new/obj/item/weapon/reagent_containers/food/snacks/grown/ambrosiadeus(src)
-			new/obj/item/weapon/flame/lighter/zippo(src)
-		if(6 to 10)
```
Comment (Sindorman) [inline code/modules/mining/abandonedcrates.dm]: Or these?

Comment (BurgerLUA) [reply code/modules/mining/abandonedcrates.dm]: because who cares

also i needed more rng slots so I decided to get rid of 2 shitty mining related drops so that kinetic accelerators may replace them

File: code/modules/mining/abandonedcrates.dm [hunk 3]
```diff
@@ -23,16 +23,10 @@
...
-			new/obj/item/weapon/pickaxe/drill(src)
-			new/obj/item/device/taperecorder(src)
-			new/obj/item/clothing/suit/space(src)
-			new/obj/item/clothing/head/helmet/space(src)
+		if(1 to 10)
```
Comment (Arrow768) [inline code/modules/mining/abandonedcrates.dm]: I think you are overdoing it a bit, you increase the dropchance tenfold.
I believe a 5% drop chance would be fine (6 to 10)
Might also be a good idea to just spawn one KA inside of the crate, instead of 3
If science is indeed unreliable, then we can explore adding medium tier KAs / KA Parts to cargo

Comment (BurgerLUA) [reply code/modules/mining/abandonedcrates.dm]: I keep forgetting to message you about this.

Yeah, I'll drop the rate down to 5%, and make some of them orderable.

**Comment by skull132** [comment]
@BurgerLUA beep.
